pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "dependency_manager"
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
this change isn't necessary. We don't change patch version on each PR and we haven't released.

Reply:
unless you explicitly do not want current users of this repo to have the latest code? That would be valid.

Reply:
Then you don't need this change at all! we do `git+https://github.com/pixee/python-dependency-manager` in that repo which means it will pull whatever is in main